Tests/Functional/Fixtures/JobManagerTest/syncJobsTable.sql

```diff
@@ -4,6 +4,6 @@ CREATE TABLE ongr_sync_jobs (id bigint(20) unsigned NOT NULL AUTO_INCREMENT,
   `document_id` VARCHAR(32) COLLATE utf8_unicode_ci NOT NULL,
   `update_type` smallint(1) NOT NULL DEFAULT '1' COMMENT '0-partial,1-full',
   `timestamp` DATETIME NOT NULL,
-  `status` tinyint(1) NOT NULL DEFAULT '0' COMMENT '0-new,1-done',
+  `status_default` tinyint(1) NOT NULL DEFAULT '0' COMMENT '0-new,1-done',
```

Comment:
Yes, it creates `ongr_sync_storage_{shop_id}` tables for every shop. But I can't find the code, which creates `ongr_sync_jobs` tables dynamically for every shop.